troubleshooting.md

```diff
@@ -429,3 +429,28 @@ You'll need to either:
 or as root if your user has not enough privileges.
 
 * sudo loginctl enable-linger $UID
+
+### 18) error creating libpod runtime: there might not be enough IDs available in the namespace
+
+Unable to pull images
+
+#### Symptom
+
+```console
+$ podman unshare cat /proc/self/uid_map
+         0       1000          1
+```
+
+#### Solution
+
+```console
+$ podman system migrate
```

Comment:
this won' be enough if there are no IDs added for the user.

I think we should document that additional IDs must be provided through the `/etc/subuid` and `/etc/subgid` files.

Reply:
What additional content would you recommend? I can provide the outputs of those files at the very least.

```
$ cat /etc/subgid 
user:100000:65536
$ cat /etc/subuid
user:100000:65536
```

Reply:
I think we can add the example you just showed, and also warn the user about overlapping ranges.  The IDs specified in these file must be unique for each user.

I'd also add a pointer to the `subuid(5)` and `subgid(5)` man pages.